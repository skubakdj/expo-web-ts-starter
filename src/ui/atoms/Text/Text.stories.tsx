import React from 'react'
import { Text } from './Text'

import { storiesOf } from '@storybook/react-native'

storiesOf('story example', module).add('just a text element', () => <Text />)
