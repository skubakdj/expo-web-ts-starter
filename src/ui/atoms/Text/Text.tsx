import React from 'react';
import { Text as TextReactNative } from 'react-native';

export const Text = () => (<TextReactNative>I am text!</TextReactNative>);

