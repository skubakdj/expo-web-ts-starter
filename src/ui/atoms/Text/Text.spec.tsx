
import 'react-native';
import * as React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import { Text } from './Text';

describe('Text', () => {
  it('jest snapshots should be working', () => {
    const tree = renderer.create(<Text />).toJSON(); 
    expect(tree).toMatchSnapshot();
  })

  it('enzyme should be working', () => {
    const wrapper = shallow(<Text />);
    expect(wrapper).toBeTruthy();
  })
})