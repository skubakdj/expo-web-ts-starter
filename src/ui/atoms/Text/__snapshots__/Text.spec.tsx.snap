// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text jest snapshots should be working 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
>
  I am text!
</Text>
`;
